commerce_coordinator/apps/commercetools/tests/raw_ct_order.json

```diff
@@ -0,0 +1,321 @@
+      "type": "centPrecision",
+      "fractionDigits": 2
+    }
+  },
+  "taxMode": "Platform",
+  "taxRoundingMode": "HalfEven",
+  "taxCalculationMode": "LineItemLevel",
+  "inventoryMode": "None",
+  "shippingAddress": {
+    "country": "US"
+  },
+  "shippingMode": "Single",
+  "shipping": [
+  ],
+  "itemShippingAddresses": [
+  ],
+  "discountCodes": [
+  ],
+  "directDiscounts": [
+  ],
+  "refusedGifts": [
+  ],
+  "paymentInfo": {
+    "payments": [
+      {
+        "typeId": "payment",
+        "id": "33766483-1428-41b0-81bb-c2ea565f7aeb",
+        "obj": {
+          "id": "33766483-1428-41b0-81bb-c2ea565f7aeb",
+          "version": 1,
+          "createdAt": "2023-10-31T19:56:36.798000+00:00",
+          "lastModifiedAt": "2023-10-31T19:56:36.798000+00:00",
+          "lastModifiedBy": {
+            "clientId": "E1UbNz9SHIG_n4nnSAUP1uW4"
+          },
+          "createdBy": {
+            "clientId": "E1UbNz9SHIG_n4nnSAUP1uW4"
+          },
+          "customer": {
+            "typeId": "customer",
+            "id": "ea421ddf-22ef-43b1-b97b-f06e5a5b8b49"
+          },
+          "interfaceId": "Stripe",
+          "amountPlanned": {
+            "centAmount": 14900,
+            "currencyCode": "USD",
+            "type": "centPrecision",
+            "fractionDigits": 2
+          },
+          "amountAuthorized": {
+            "centAmount": 14900,
+            "currencyCode": "USD",
+            "type": "centPrecision",
+            "fractionDigits": 2
+          },
+          "amountPaid": {
+            "centAmount": 14900,
+            "currencyCode": "USD",
+            "type": "centPrecision",
+            "fractionDigits": 2
+          },
+          "paymentMethodInfo": {
+            "paymentInterface": "Stripe",
+            "method": "credit card",
+            "name": {
+              "en": "Mastercard",
+              "en-US": "Mastercard"
+            }
+          },
+          "paymentStatus": {
+            "interfaceCode": "completed",
+            "interfaceText": "Completed"
+          },
+          "transactions": [
+            {
+              "id": "db34acf7-2022-40b1-97f4-fbaf71acba4f",
+              "type": "Charge",
+              "amount": {
+                "centAmount": 14900,
+                "currencyCode": "USD",
+                "type": "centPrecision",
+                "fractionDigits": 2
+              },
+              "state": "Initial"
+            }
+          ],
+          "interfaceInteractions": [
+          ]
+        }
+      }
+    ]
+  },
+  "country": "US",
+  "origin": "Customer",
+  "cart": {
+    "typeId": "cart",
+    "id": "b19d2a92-3a9a-449a-aa2a-4118f13f09ad"
+  },
+  "orderState": "Complete",
+  "shipmentState": "Shipped",
+  "paymentState": "Paid",
+  "syncInfo": [
+  ],
+  "returnInfo": [
+  ],
+  "lastMessageSequenceNumber": 7,
+  "lastModifiedBy": {
+    "clientId": "E1UbNz9SHIG_n4nnSAUP1uW4"
+  },
+  "createdBy": {
+    "clientId": "E1UbNz9SHIG_n4nnSAUP1uW4"
+  }
+}
```

Comment:
EOF line missing - might error our in linting

Reply:
Fixed in f7c34d46ddfec481c86b2ca24872cb1b460d7f72!